content/docs/get-started/serving.md

```diff
@@ -31,10 +34,37 @@ That's it! You can check out the other available server implementations in
 
 ## Making requests
 
-While serving the model, you can open Swagger UI (OpenAPI) at
-[http://localhost:8080/docs](http://localhost:8080/docs) to check out OpenAPI
-spec and query examples.
+While the model server is running, you can use your browser to open the Swagger
+UI (OpenAPI) at [http://localhost:8080/docs](http://localhost:8080/docs) to
+check out OpenAPI spec and query examples.
+
+Or, alternatively, launch requests at the server from a different terminal,
+Using `curl` or `httpie`. For example:
 
-MLEM can help you with not only serving, but also querying the model. Using MLEM
-API or CLI you can send requests to your model without implementing the "client"
-part. Learn more about this in [User Guide](/doc/user-guide/serving/).
+```cli
+$ curl -X 'POST' \
+  'http://0.0.0.0:8080/predict_proba' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "data": {
+    "values": [
+      {
+        "sepal length (cm)": 0,
+        "sepal width (cm)": 1,
+        "petal length (cm)": 2,
+        "petal width (cm)": 3
+      }
+    ]
+  }
+}'
+[[0.47,0.24,0.29]]
+```
+
+The Swagger UI even helps to generate `curl` commands similar to the above
+interactively.
```

Comment:
by proxy yes, because you get swagger UI, and it has some nice conveniences that people which aren't familiar with it may not know just by mentioning the name 🤔

Reply:
We are not selling Swagger UI here, I think. Users can open the link and see it, if they follow the GS. Plus, many should be already familiar with it. So, I'm for removing this.

Reply:
Ah, ok, it's hidden under the collapsible part. Then it's ok I think.